sphinxemoji/sphinxemoji.py

```diff
@@ -90,8 +92,13 @@ def copy_asset_files(app, exc):
 def setup(app):
     app.connect('build-finished', copy_asset_files)
     style = app.config._raw_config.get('sphinxemoji_style')
+    source = app.config._raw_config.get('sphinxemoji_source')
     if style in emoji_styles:
-        for fname in emoji_styles[style]:
+        if source is None:
+            source = emoji_styles[style]['source']
+        app.add_js_file(source)
+        libs = emoji_styles[style]['libs']
+        for fname in libs:
```

Comment:
Maybe you can:

```
source = app.config._raw_config.get('sphinxemoji_source', emoji_styles[style]['source'])
```

Can you add a quick note in the README file about this? (the new parameter and the CDN)

Also:

```
for fname in emoji_styles[style]['libs']
```

Reply:
Thx for the feedback. I think i addressed all of your comments. Can you take a look?